### DISCUSSION TITLE
Job opportunities confusion

### DISCUSSION THREAD

**Discussion by Prajwal0822** [snigi-gupta/AttendingUniversityAtBuffalo#15] [Q&A] [answered]
Is work experience important before masters?
Or directly after Bachelors can we pursue masters.
Will we get a job after graduation.

**Comment by snigi-gupta** [comment, marked as answer]
@Prajwal0822 ,

Firstly I will suggest you read the [Guidelines](https://gist.github.com/snigi-gupta/90b2ba5fa4d98eb4f529c72e146e0fef).

Now to answer about experienced student studying MS vs newgrad student studying MS:
- If you are experienced (1-3+ yrs of experience), then that will bring some advantage. I say some because it only gets you the opportunity. During the interview process, they also interview you accordingly, and the interviews then get tough.
- If you are not experienced, then getting an opportunity can be slightly be tough, but you will get the job if you are doing all the right things.
- The right things are : Working hard, practicing, networking, collaborating, volunteering, community contributions, research/internships/self-projects/thesis, not cheating/plagiarizing, **working on your resume**.
- If your current conditions/age/finances allow you to gain some experience and then come, then my personal suggestion is to defer.
- If you cannot defer for XYZ reasons, then also there nothing to worry about. If Class of 2021 can secure jobs during an on-going pandemic, a very bad recession, Asian hate, then I think anyone can.
- TBH jobs/internships all depend on you your hard work and smart work. You need both here in the USA. The system here is very different from India. 

I will soon post some info, tips, and tricks that can help you a tad bit. Make sure you have this repo starred/followed. 

Let me know if I answered your question.
